agent/php_execute.c

```diff
@@ -928,6 +952,21 @@ static void nr_execute_handle_logging_framework(const char* filename,
 
 #undef STR_AND_LEN
 
+static void nr_execute_handle_package(const char* filename) {
+  char* filename_lower = nr_string_to_lowercase(filename);
```

Comment:
if `filename` is `null` `nr_string_to_lowercase` will return a `0`. You may want to add a `null` check as well as check string length > 0 check before calling this function.

Reply:
Fixed: [be9c572](https://github.com/newrelic/newrelic-php-agent/pull/775/commits/be9c5728e2763a3c0546ecd9ed8930fb0599d4cf)